### ISSUE TITLE
x/tools/gopls: no documentation on hover for package-level variables and constants

### ISSUE THREAD

**Issue by inliquid** [golang/go#33793]
1. VS Code
2. master branch of `gopls`
```
>gopls version
golang.org/x/tools/gopls v0.1.3
    golang.org/x/tools/gopls@(devel)
```

No documentation is provided on hover for package-level variables and constants, f.e.:
![изображение](https://user-images.githubusercontent.com/6797881/63542006-fc5f2100-c527-11e9-82fb-d5fcc8a2dbb9.png)

**Comment by srikrsna** [comment]
I'm interested in addressing this. I am very new to this can someone guide me?

**Comment by JAicewizard** [comment]
This cant be fixed ATM because in the AST the comment is outside of the variable declaration and part of the var expression.

**Comment by ridersofrohan** [comment]
Fixed in [#004141d](https://github.com/golang/tools/commit/acc15743c3f656ff416c076db65639213ff94cb8)

![Screen Shot 2019-11-25 at 13 31 13](https://user-images.githubusercontent.com/3170854/69567655-dfebde00-0f87-11ea-8a30-416bfa7d04be.png)
